### ISSUE TITLE
Players will only shoot inside foot shot / chip shot

### ISSUE THREAD

**Issue by lucvanloon** [xAranaktu/FIFA-19---Career-Mode-Cheat-Table#18]
After editing Wayne Rooney and Robin van Persie (who I club transferred before starting career mode) they now only shoot with the inside foot shot. The shot is also not very powerful.

They also both have the chip shot trait, which I don't think I can turn off. So maybe it's a chip shot.

I can shoot fine with other players. Is there something I'm editing that does this?

**Comment by xAranaktu** [comment]
I don't know what exactly you have changed. It may be caused by one of the trait. Traits can be edited in '"Database tables" -> "players table". Try to remove all of them.

**Comment by lucvanloon** [comment]
Thanks for your reply. Rooney for example has 3 traits: Long Passer, Long shots and chip shot. I can't remove those from the traits options at all. I've got all set to none for him.

Looks like I can't really fix it, I don't know how. For Rooney I edited his potential and age.

If you wanna take a look here are my career files (it's the Man United one in game):
https://www.mirrored.to/files/0VKCIGXZ/settings.zip_links
or wetransfer link: https://we.tl/t-SXUSLw2W3M
